File: UETshare/src/main/java/com/example/uetshare/response/dto/NotificationQuestionDto.java
package com.example.uetshare.response.dto;

import com.example.uetshare.entity.ActionType;
import com.example.uetshare.entity.Question;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Calendar;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class NotificationQuestionDto {

    private Long id;
    private Long question_id;
    private String username;
    private String avatar;
    private Boolean seen;
    private ActionType action_type;
    private Calendar time;
    private String owner_username;
}

File: app/src/main/java/ie/app/uetstudents/Entity/notifications_question/put/respont/NotificationQuestionDto.kt
package ie.app.uetstudents.Entity.notifications_question.put.respont

data class NotificationQuestionDto(
    val action_type: String,
    val avatar: String,
    val id: Int,
    val question_id: Int,
    val seen: Boolean,
    val time: String,
    val username: String
)